### ISSUE TITLE
Create an endowment (for specific, known expenses)

### ISSUE THREAD

**Issue by ariddell** [pybee/paying-the-piper#47]
The Stanford Encyclopedia of Philosophy (SEP) is volunteer driven but has ongoing costs, just like Django. It seems to have succeeded in raising money to support itself via an endowment. One feature of this endowment is interesting, if the SEP ever shuts down, those who gave money to the endowment will receive their donation back. Moreover, I think interest from the endowment can only be used to pay for a specific range of things. Both these features get at what @jacobian is talking about in #43: donors need to _feel confident that their money is being spent effectively_. Endowments where the range of spending is restricted to expenditures of obvious utility are one established way to address this concern.

Details:
- http://sca.jiscinvolve.org/wp/portfolio-items/stanford-encyclopedia-of-philosophy-building-an-endowment-with-community-support-stanford-university-california-usa-ithaka-case-studies-in-sustainability/

Coverage:
- http://qz.com/480741/this-free-online-encyclopedia-has-achieved-what-wikipedia-can-only-dream-of/ (goes into some detail about the endowment)
- http://www.neh.gov/humanities/2011/januaryfebruary/feature/homepage-philosophy

**Comment by freakboy3742** [comment]
I'm not entirely sure how the SEP model works here. From the QZ article:

> To be fair, Stanford does pay most of the operating costs. But the SEP has a paid staff of only three—Zalta, Nodelman, and Allen—plus five other Stanford employees who spend 20% of their time on technical support. Neither the authors, nor the dozens of subject editors, get so much as a dime for their troubles.

So... yes, given a magical pot of money to pay for management, and an endless supply of free volunteers, it's possible to run a project. We already know this :-) 

The endowment angle is interesting - although as practiced by SEP it seems to be more like a freemium/razor blades model - they give the content away for free, but sell (by way of endowment) premium features like whiteboxing and fancy printed versions:

> Libraries were enthusiastic. The SEP was able to raise over $2 million from the long list of contributors, and Stanford added $1 million to the library endowment. The university also provides 60% of SEP’s budget—not much to ask from such a rich institution. The remaining 10% comes from a “friends of the SEP” program, which for $5, $10, or $25 a year lets individual users download nicely formatted PDFs of the articles, good for printing or archiving for personal use.

They also, clearly, get a lot of money from a benevolent donor - Stanford University itself.

But as a model for how to attract sponsors - a clearly defined goal, with the possibility of paying back some capital - it may be worth exploring. It makes the funding goal higher (since you are essentially aiming to pay costs out of interest, rather than principle), but pitching as a "low yield investment with a social good upside" may make it easier to sell as an idea..

**Comment by ariddell** [comment]
I could see something like this: the endowment is set aside to pay for N/hours a week of programming time for tasks deemed critical by a majority of the DSF board. If, in any given week, it is not spent doing this it goes back into the endowment. Maybe this is too much money to raise? If so, the exercise in calculating the cost would be a useful way of calibrating thinking about paying-the-piper.

**Comment by nayafia** [comment]
Endowments are great models for stuff like this, the question is how much do you need and who supports it.

A fair amount of SEP's operating costs are still being underwritten by Stanford. From what I can tell from the QZ article, their endowment looks something like this:

$2M donated from ~550 academic libraries (roughly $3.5K each, not counting partial contributors)
$1M donated from Stanford
= $3M endowment

Endowments spend ~5% per year, so that gives them an annual budget of $150K/yr. This is apparently 1/3 of their budget. 60% comes from Stanford and 10% from indiv contributors. So their annual budget looks something like this:

$150K from endowment
$300K from Stanford
$50K from individual contributions ("friends of SEP")
= $500K/yr budget

@ariddell I think you get to an interesting point: how much money is actually needed to support something like Django? What are the ballpark annual operating costs in an ideal world? Having a concrete funding goal can help us figure out whether something like an endowment is feasible or not.

**Comment by ariddell** [comment]
I suppose $500K/yr might be right. It seems high. Although "[T]he SEP has a paid staff of only three—Zalta, Nodelman, and Allen—plus five other Stanford employees who spend 20% of their time on technical support."

I suspect Django could raise $1M judging from Ruby Together's success. I suppose one idea here is that there might be different funding strategies for different kind of work. I could see the attraction (for donors) of funding an endowment to fund maintenance and security work. I bet you could raise more money in a shorter period of time for that cause than for other development work.

**Comment by freakboy3742** [comment]
@ariddell It might be different in the long term, but at least for the moment, the amount of work that needs to be done vastly exceeds the available funds. I don't think a refund for not hitting an "N hours" target is very likely.

**Comment by freakboy3742** [comment]
@nayafia The question of "how much the DSF would need" is a bit like "how long is a piece of string" - the more money we have, the more we can do.

The bare minimum at the moment is enough to pay for one DSF fellow, full time. That's currently running to about $80k/year. However, the current fellow (@timgraham) is currently doing the DSF a bit of a favour, taking a not-insignificant pay cut in the interests of the broader community. Long term, he shouldn't have to do that. We might be able to find someone who is cheaper - but if we want to maintain engineering quality, we can't just offshore to the cheapest bidder. So, we're probably looking for at least a $120-150k minimum annual budget. 

However, if we can get more, we can expand our operations to include other key projects (like Django REST Framework, or Django Registration, or other important pieces of infrastructure). 

My mental model of the ideal case would be 6-10 people, ranging from designers to operations, ranging in experience from intern to senior engineer, and someone doing admin. A rough calculation means we'd be looking at an annual budget of $750k-$1.5m.

**Comment by nayafia** [comment]
Yeah, that would require a pretty big endowment to sustainably cover an annual budget of that size ($15M for a $750K annual budget). We should take this offline sometime, but since Django qualifies as a "big project" and not a "smallish project", I'm curious what you've tried thus far around getting big institutional funders in the door!

**Comment by freakboy3742** [comment]
Agreed that it's a high bar. I'll shoot you an email about the details of Django's specific case.
